controls/raddaterangepicker/programmatic-selection.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Programmatic Selection
+page_title: Programmatic Selection
+description: Check our &quot;Programmatic Selection&quot; documentation article for the RadDateRangePicker WPF control.
+slug: raddaterangepicker-programmatic-selection
+tags: daterangepicker,selection,range
+published: True
+position: 9
+---
+
+# Programmatic Selection
+
+The date range can be selected through the UI or via the available API of the DateRangePicker component.
+
+The selection can be assigned manually via the `StartDate` and `EndDate` properties of `RadDateRangePicker`.
+
+#### __[XAML] Setting the selection range in XAML__
+{{region raddaterangepicker-programmatic-selection-0}}
+	<telerik:RadDateRangePicker StartDate="2024/1/19" EndDate="2024/2/5" />
+{{endregion}}
+
+#### __[XAML] Setting the selection range in C#__
+{{region raddaterangepicker-programmatic-selection-1}}
+	this.dateRangePicker.StartDate = new DateTime(2024, 1, 19);
+	this.dateRangePicker.EndDate = new DateTime(2024, 2, 5);
+{{endregion}}
+
+![Picture showing WPF RadDateRangePicker with assigned selection range](images/raddaterangepicker-programmatic-selection-0.png)
+
+## Defining Available Selection Range
+
+The dates available for selection can be limited via the `SelectableDateStart` and `SelectableDateEnd` properties of `RadDateRangePicker`. Setting this will disable all other dates in the calendars.
+
+#### __[XAML] Setting the selection range in XAML__
```

Comment:
Shouldn't the code snippets show the SelectableDateStart and SelectableDateEnd properties, instead of showing the StartDate and EndDate ones?